plugin/adsk/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ set(INSTALL_DIR_SUFFIX plugin/adsk)
 # Packages 
 #==============================================================================
 add_subdirectory(plugin)
-
+add_subdirectory(scripts)
```

Comment:
I added the maya usd proxy shape template below. This fixes a layout error in the AE (because of missing scroll layout).